### ISSUE TITLE
TypeScript Error on webhookConfig stripeSecrets

### ISSUE THREAD

**Issue by MuttakinHasib** [golevelup/nestjs#554]
![image](https://user-images.githubusercontent.com/44552983/217545874-71044817-b1eb-4880-8578-8f6fa095a7c1.png)

@WonderPanda Please fix this ASAP

**Comment by MuttakinHasib** [comment]
@underfisk  please check it

**Comment by marcalj** [comment]
For some reason the types are not recognised, since the config object definition is this:

```ts
export interface StripeModuleConfig extends Partial<Stripe.StripeConfig> {
    readonly apiKey: string;
    /**
     * Configuration for processing Stripe Webhooks
     */
    webhookConfig?: {
        /**
         * The webhook secret registered in the Stripe Dashboard
         */
        stripeWebhookSecret: string;
        /**
         * The property on the request that contains the raw message body so that it
         * can be validated. Defaults to 'body'
         */
        requestBodyProperty?: string;
        /**
         * The prefix of the generated webhook handling controller. Defaults to 'stripe'
         */
        controllerPrefix?: string;
        /**
         * Any metadata specific decorators you want to apply to the webhook handling controller.
         *
         * Note: these decorators must only set metadata that will be read at request time. Decorators like Nest's `@UsePipes()` or `@UseInterceptors()` wll not work, due to the time at which Nest reads the metadata for those, but something  that uses `SetMetadata` will be fine, because that metadata is read at request time.
         */
        decorators?: ClassDecorator[];
        /**
         * Logging configuration
         */
        loggingConfiguration?: {
            /**
             * If enabled will log information regarding event handlers that match incoming webhook events
             */
            logMatchingEventHandlers: boolean;
        };
    };
}
```

In Webstorm, the `.forRoot` method is highlighted in error as: `Instance member is not accessible `

**Comment by MuttakinHasib** [comment]
https://www.npmjs.com/package/@golevelup/nestjs-stripe

@marcalj **they didn't publish yet the npm package but they have updated on their source code on Github**


![image](https://user-images.githubusercontent.com/44552983/220799172-c2d3231e-2fc6-4e13-bcdf-787d44cd86c5.png)

**Comment by underfisk** [comment]
I'm sorry for getting back so late on the topic
@MuttakinHasib is right, we do have the new interface's update but only @WonderPanda has permissions to publish a new version

**Comment by WonderPanda** [comment]
Apologies guys. The latest is published now with `@golevelup/nestjs-stripe@0.6.0`

**Comment by MuttakinHasib** [comment]
Thanks @underfisk @WonderPanda
